Mathlib/Analysis/SpecificLimits/Normed.lean

```diff
@@ -214,6 +214,18 @@ theorem tendsto_pow_const_mul_const_pow_of_abs_lt_one (k : ℕ) {r : ℝ} (hr :
   rw [tendsto_zero_iff_norm_tendsto_zero]
   simpa [div_eq_mul_inv] using tendsto_pow_const_div_const_pow_of_one_lt k hr'
 
+lemma tendsto_const_div_atTop {F : Type*} [TopologicalSpace F] [LinearOrderedField F]
+    [OrderTopology F] (g : ℕ → F) (r : F) (hg : Tendsto g atTop atTop) :
+    Tendsto (fun n ↦ r / g n) atTop (𝓝 0) := by
+  apply Filter.Tendsto.div_atTop tendsto_const_nhds hg (f := fun _ => r) (a := r)
```

Comment:
This still isn't general enough: it should have a generic type in place of `ℕ` and a generic filter in place of `atTop` (on `ℕ`). Also, it needs to be moved just after `Filter.Tendsto.div_atTop`.